### ISSUE TITLE
step4 - Point mapper fine-tuning

### ISSUE THREAD

**Issue by sallysooo** [sallysooo/AERO#5]
## Algorithm 2.4 : pointmapper fine-tuning

<img width="1606" height="748" alt="Image" src="https://github.com/user-attachments/assets/2e875bb7-16b1-4ae5-ae96-ff4f2570d692" />

## To do List
> In this step, we finally fine-tune the pretrained pointmapper.
- [x] Make the fine-tuning logic referring to the loss function equation (5) below. (same as equation (3), except for the fact that the volatile m is replaced with a fixed `a` while considering the MSE)
- [x] Derive epoch3 through this step.

<img width="1345" height="211" alt="Image" src="https://github.com/user-attachments/assets/7b18fc7e-a58e-4d63-accd-6352701ca125" />
